Intersect (Core)/Network/PackedIntersectPacket.cs

```diff
@@ -0,0 +1,60 @@
+﻿using MessagePack;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Intersect.Network
+{
+    [MessagePackObject]
+    public class PackedIntersectPacket
+    {
+        private static readonly string[] BuiltInPacketNamespaces = new[]
+        {
+            "Intersect.Network.Packets",
+            "Intersect.Network.Packets.Client",
+            "Intersect.Network.Packets.Editor",
+            "Intersect.Network.Packets.Server",
+            "Intersect.Admin.Actions"
+        };
+
+        public static Dictionary<Type, Int32> KnownTypes { get; set; } = new Dictionary<Type, Int32>();
+
+        public static Dictionary<Int32, Type> KnownKeys { get; set; } = new Dictionary<Int32, Type>();
```

Comment:
`int`

Reply:
Meant to do an Int16 ---- changed to just short. (Keeping packet sizes small, assuming this matters for messagepack)